### PR TITLE
Corrected paragraph formatting

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request
 
This pull request (PR) fixes issue #58 
 
## Affected Support KB pages
 
https://support.magento.com/hc/en-us/articles/360059785811

### CODE REVIEW

File: src/support-tools/patches-available-in-mqp-tool/mdva-19640-magento-patch-advanced-reporting-shows-no-data.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ title: "MDVA-19640 Magento patch: Advanced reporting shows no data"
 labels: 2.3.1,2.3.2,2.3.2-p2,2.3.3,2.3.3-p1,2.3.4,2.3.4-p1,2.3.4-p2,2.3.5,2.3.5-p1,2.3.5-p2,2.3.6,2.3.6-p1,2.4.0,2.4.0-p1,2.4.1,2.4.1-p1,2.4.1-p2,2.4.2,Advanced Reporting,MQP 1.0.20,MQP patches,Magento Commerce,Magento Commerce Cloud,empty,no data
 ---
 
-The MDVA-19640 Magento patch fixes the issue when Advanced Reporting shows no data.
-
-This patch is available when the [Magento Quality Patch (MQP) tool](https://support.magento.com/hc/en-us/articles/360047139492) 1.0.20 is installed. The patch ID is MDVA-19640. Please note that there is no current plan to fix this issue in future versions.
+The MDVA-19640 Magento patch fixes the issue when Advanced Reporting shows no data. This patch is available when the [Magento Quality Patch (MQP) tool](https://support.magento.com/hc/en-us/articles/360047139492) 1.0.20 is installed. The patch ID is MDVA-19640. Please note that there is no current plan to fix this issue in future versions.
```
Comment (jmenn) [inline src/support-tools/patches-available-in-mqp-tool/mdva-19640-magento-patch-advanced-reporting-shows-no-data.md:6]:
```suggestion
The MDVA-19640 Adobe Commerce patch fixes the issue when Advanced Reporting shows no data. This patch is available when the [Magento Quality Patch (MQP) tool](https://support.magento.com/hc/en-us/articles/360047139492) 1.0.20 is installed. The patch ID is MDVA-19640. Please note that there is no current plan to fix this issue in future versions.
```

**Review by jmenn — CHANGES_REQUESTED** [review]

File: src/support-tools/patches-available-in-mqp-tool/mdva-19640-magento-patch-advanced-reporting-shows-no-data.md (hunk 1 above)
Comment (BarnyShergold) [reply src/support-tools/patches-available-in-mqp-tool/mdva-19640-magento-patch-advanced-reporting-shows-no-data.md:6]: The problem here is that "Adobe Commerce" is the licensed version. The free version is "Magento Open Source" - making the change above implies that it is only for the version. In this case Magento is generic.

**Comment by BarnyShergold** [comment]
@jmenn - I've raised the use of "Adobe Commerce", "Magento Open Source" & "Magento" in the KB channel - I'm concerned that only using "Adobe Commerce" could imply that it's only for the licensed version. Just want to an agreed view on this.

**Review by jmenn — APPROVED** [review]

**Comment by jmenn** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by magento-engcom-team** [comment]
@jmenn the branch with code successfully imported into`magento-kb/knowledge-base` repository. Branch name: `imported-magento-knowledge-base-74`.

**Comment by jmenn** [comment]
Hi Barny!

This PR got caught up in part of a bulk rebrand, so it conflicts now.

Can you close this PR and make a new PR for this file with these things below replaced, please?

The rebrand changed these things:

MQP->QPT
Magento Quality Patches tool -> Quality Patches Tool
Magento Quality Patches ->Quality Patches Tool
Magento Quality Patch ->Quality Patches Tool
Quality Patches Tool (QPT) tool -> Quality Patches Tool (QPT)

Let me know and then I'll approve it, and do the steps to get it merged..... Sasha knows about this instance, also.  Thanks!

**Comment by BarnyShergold** [comment]
Closed to re-open on another branch to solve complex conflicts

**Pull request closed without merging**